README.md

```diff
@@ -3,6 +3,8 @@
 
 [![Join the chat at https://gitter.im/VirtusLab/git-machete](https://badges.gitter.im/VirtusLab/git-machete.svg)](https://gitter.im/VirtusLab/git-machete?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
```

Comment:
Nit: remove the empty line so that the badges are next to each other rather than one below another (at least that's how they're usually aligned on most OSS projects IIRC)

Reply:
OK.